cpp/src/sampling/random_walks.cuh

```diff
@@ -1003,18 +1031,27 @@ struct coo_convertor_t {
  * @brief returns random walks (RW) from starting sources, where each path is of given maximum
  * length. Uniform distribution is assumed for the random engine.
  *
- * @tparam graph_t Type of graph (view).
+ * @tparam graph_t Type of graph/view (typically, graph_view_t).
```

Comment:
What is "/"?

Reply:
It means: "graph" or "graph_view" => graph/graph_view => (by contraction) "graph/view".